src/server/active_tran_server.cpp

```diff
@@ -183,17 +185,22 @@ active_tran_server::init_page_server_hosts (const char *db_name)
       const int local_exit_code = connect_to_page_server (node, db_name);
       if (local_exit_code == NO_ERROR)
 	{
-	  // found valid host clear the errors from the bad ones
 	  ++valid_connection_count;
 	  exit_code = NO_ERROR;
+	  if (!connected)
+	    {
+	      first_valid_connection = node;
+	    }
+	  //found valid host clear the errors rom the bad ones
 	  er_clear ();
-	  // successfully connected to a page server. stop now.
-	  break;
+	  connected = true;
+	  continue;
```

Comment:
remove this `continue` by moving the `er_log_debug..` on the else branch below this comment

Reply:
done. was old code relics